resources/public/api/conf/1.0/lossClaims_amend.raml

```diff
@@ -0,0 +1,25 @@
+is:
+  - acceptJson
+  - contentTypeJson
+  - testScenarioHeader
+  - formatNino
+  - formatClaimId
+  - incorrectOrEmptyBody
+  - formatTypeOfClaim
+  - ruleTypeOfClaim
+  - notFound
+displayName: Amend Loss Claims
+description: This endpoint allows the user to update existing loss claims.
+(annotations.sandboxData): !include scenarios/lossClaims_amend.md
+(annotations.scope): "write:self-assessment"
+securedBy: [ sec.oauth_2_0: { scopes: [ "write:self-assessment" ] } ]
+body:
+  application/json:
+    type: !include schemas/lossClaims/amendClaims/amend_request.json
+    example: !include examples/lossClaims/amendClaims/amend_request.json
+responses:
+  201:
```

Comment:
Should this be a 204?

Reply:
We don't have a tech spec yet so I'm not sure. The Amend BF endpoint is a 201 though so I went with that for now.